src/plugins/intel_gpu/src/plugin/transformations/lora_horizontal_fusion.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+
+namespace ov {
+namespace intel_gpu {
+
+class LoRAHorizontalFusion: public ov::pass::MatcherPass {
```

Comment:
Please add a diagram in a comment which describes the transformation as the current pattern code is not so trivial